### DISCUSSION TITLE
Problem running implicit run in OpenRadioss

### DISCUSSION THREAD

**Discussion by Semalty** [OpenRadioss/OpenRadioss#923] [Q&A]
Hi all. I am trying to run an implicit analysis using OpenRadioss but I am getting the error, "impi.dll not found, reinstalling might solve the problem". May anyone please tell me how to solve the problem? I downloaded the compiled binaries from Github.

Regards,
Rachit

**Comment by sebastienVilleneuve** [comment]
Hi @Semalty 

did you install the intel mpi ?
![image](https://user-images.githubusercontent.com/100404030/208918334-f8513eb9-b874-4eed-8b1a-8f5cb7e7d525.png)

Thanks,
Sébastien

**Reply by Semalty** [reply]
Hi Sebastian. Thanks for your reply. No, I have not installed intel MPI. I have downloaded the files you mentioned in the comments but installing them will take some time as I will need the company's approval to install them on the system. 

OpenRadioss download folder has an "IntelOneAPI_runtime" folder. I have added it to the path variable. There was no problem running the explicit analysis model but I am not able to run the implicit analysis model. 

Am I required to change the value of the path variable to the OneAPI install folder after it is completed or will I just need to take some files from the install folder and place them in the "IntelOneAPI_runtime" folder?

**Reply by laurent-altr** [reply]
Hi, 
Implicit feature is not yet available on native Windows. 
A workaround is to use WSL. Anyway, implicit feature requires an additional library that we do not provide (MUMPS). You will have to compile it on your own (see the help of the build_script).

Your error does not seem specific to implicit though, are you able to run explicit analysis?
Thank you, 
Laurent

**Reply by Semalty** [reply]
Hi Laurent. Yes, I am able to run explicit analysis. I will try to compile the MUMPS library. Is there no other way of solving the implicit analysis problem in OpenRadioss?

Regards

Rachit

**Reply by OlivierWienholtz** [reply]
Hello, 

No there is not other way to Solve implicit. You need Mumps.

Regards,
Olivier

**Reply by qiangRadioss** [reply]
Iterative solver could be used w/ implicit solution (see /IMPL/SOLVER), but it's not recommended, it generally takes much more computation times and not as robust as direct solver like MUMPS. 

Regards.
Qiang.
